nupic/torch/modules/k_winners.py

```diff
@@ -249,6 +248,11 @@ def __init__(
         :param duty_cycle_period:
           The period used to calculate duty cycles
         :type duty_cycle_period: int
+
+        :param local:
+            Whether or not to choose the k-winners locally across the channels
```

Comment:
`...locally (across the channels at each location) or globally (across the whole input and across all channels).`